src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -109,6 +112,35 @@ public void tearDown() throws Exception {
     jobIds.forEach(this::deleteJob);
   }
 
+  @Test
+  public void enumerateTooLongBackFilterTest() throws Exception {
+    List<JobModel> expectedJobs = new ArrayList<>();
+    int numVisibleJobs = 3;
+    for (int i = 0; i < numVisibleJobs; i++) {
+      String jobId = runFlight(makeDescription(i), makeFlightClass(i), testUser);
+      expectedJobs.add(
+          new JobModel()
+              .id(jobId)
+              .jobStatus(JobStatusEnum.SUCCEEDED)
+              .statusCode(HttpStatus.I_AM_A_TEAPOT.value())
+              .description(makeDescription(i))
+              .className(makeFlightClass(i).getName()));
+    }
+    assertThat(
+        "All jobs are returned",
+        jobService.enumerateJobs(0, 100, testUser, SqlSortDirection.ASC, ""),
+        contains(getJobMatchers(expectedJobs)));
+
+    // Update the submission time of the first job to be a long time ago
+    updateJobSubmissionTime(
+        expectedJobs.get(0).getId(),
+        Instant.now().minus(JobService.DAYS_BACK_TO_QUERY + 1, ChronoUnit.DAYS));
```

Comment:
If this was using `Duration` you could write
```suggestion
        Instant.now().minus(JobService.TIME_BACK_TO_QUERY).plus(Duration.ofDays(1)));
```